odl/util/phantom.py

```diff
@@ -410,6 +502,10 @@ def shepp_logan(space, modified=False):
     shepp_logan(discr, modified=False).show()
     derenzo_sources(discr).show()
 
+    discr = odl.uniform_discr([-2.5, -2.5], [2.5, 2.5], [n, n])
```

Comment:
This is prolly the only "real" complaint. These phantoms should (imo) work on arbitrary domains, i.e. not only ones that fit in `[-2.5, -2.5] x [2.5, 2.5]`. You may want to rescale the points so this works. This is how the other phantoms work.